omnigibson/systems/micro_particle_system.py

```diff
@@ -1567,8 +1567,13 @@ def clothify_mesh_prim(cls, mesh_prim, remesh=True, particle_distance=None):
                 cloth particles are roughly touching each other, given cls.particle_contact_offset and
                 @mesh_prim's scale
         """
-        # Possibly remesh if requested
-        if remesh:
+        has_uv_mapping = mesh_prim.GetAttribute("primvars:st").Get() is not None
+        if not remesh:
+            # We always load into trimesh to remove redundant particles (since natively omni redundantly represents
+            # the number of vertices as 6x the total unique number of vertices)
+            tm = mesh_prim_to_trimesh_mesh(mesh_prim=mesh_prim, include_normals=True, include_texcoord=True, world_frame=False)
```

Comment:
did we validate that this works? i don't remember why we needed the world frame thing either

Reply:
discussed offline.